### DISCUSSION TITLE
I want to know what each options mean and how to migrate from `poetry.toml` to `pyoproject.toml`.

### DISCUSSION THREAD

**Discussion by 0417taehyun** [python-poetry/poetry#7361] [Q&A] [answered]
I know that since version [0.3.0](https://github.com/python-poetry/poetry/releases/tag/0.3.0), poetry supports `pyproject.toml` as default. However, there was the old version of `poetry.toml` file on my project and I want to migrate it to `pyproject.toml` file. The old one looks like the one below.

```TOML
[virtualenvs]
create = true
in-project = true
path = ".venv"
```

I search the document to know each configuration option meaning. However, I could not find them.

Are there anyways to know what each options mean and how to migration them to the `pyproject.toml` file safely?

**Comment by radoering** [comment, marked as answer]
`poetry.toml` is a file for config settings that is still valid. See https://python-poetry.org/docs/configuration/

https://python-poetry.org/docs/configuration/#virtualenvscreate
https://python-poetry.org/docs/configuration/#virtualenvsin-project
https://python-poetry.org/docs/configuration/#virtualenvspath

**Reply by 0417taehyun** [reply]
Thanks!
